src/lib/y2network/connection_configs_collection.rb

```diff
@@ -53,6 +53,14 @@ def by_name(name)
       connection_configs.find { |c| c.name == name }
     end
 
+    # Returns a connection configuration tied to the given interface
+    #
+    # @param iface [Y2Network::Interface]
+    # @return [ConnectionConfig, nil] Connection config tied to the interface or nil if not found
+    def by_interface(iface)
+      connection_configs.find { |c| c.interface == iface }
```

Comment:
with profiles it can be more then one, not?

Reply:
eh ... this function is not used at all - i tried to create something and in the end went by another way ... I will drop it.